globus_cli/parsing/shared_options.py

```diff
@@ -1,10 +1,35 @@
 import click
 
-from globus_cli.version import __version__
+from globus_cli.version import get_versions
 from globus_cli.parsing.command_state import format_option
 from globus_cli.parsing.case_insensitive_choice import CaseInsensitiveChoice
 
 
+def version_option(f):
+    """
+    Largely a custom clone of click.version_option -- almost identical, but
+    makes more assumptions and prints our special output.
+    """
+    def callback(ctx, param, value):
+        latest, current = get_versions()
+        click.echo(('Installed Version: {0}\n'
+                    'Latest Version:    {1}\n'
+                    '\n{2}').format(
+                        current, latest,
+                        'You are running the latest version of the Globus CLI'
+                        if current == latest else
+                        'You should update your version of the Globus CLI'
```

Comment:
Maybe throw a link to https://globus.github.io/globus-cli/#updating-removing-the-cli here so people know how.